core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -365,6 +365,27 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 		return errors.Errorf("invariant violation: expected transaction %v to be in_progress, it was %s", etx.ID, etx.State)
 	}
 
+	lgr := eb.logger.With(
+		"ethTxID", etx.ID,
+		"checker", etx.TransmitChecker,
+		"gasLimit", etx.GasLimit,
+		"gasPrice", attempt.GasPrice,
+		"gasTipCap", attempt.GasTipCap,
+		"gasFeeCap", attempt.GasFeeCap,
+	)
+	if meta, err := etx.GetMeta(); err == nil && meta != nil {
+		lgr = lgr.With(
+			"jobID", meta.JobID,
+			"upkeepID", meta.UpkeepID,
```

Comment:
would this default to 0 for non keeper jobs? Is there a way we can avoid adding this for non keeper jobs

Reply:
Good question. Upkeep ID starts from 0 so I'm not sure. Let me think about it.

Reply:
Fixed